src/quo2/screens/navigation/bottom_nav_tab.cljs

```diff
@@ -0,0 +1,61 @@
+(ns quo2.screens.navigation.bottom-nav-tab
+  (:require [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [reagent.core :as reagent]
+            [quo2.components.navigation.bottom-nav-tab :as quo2]
```

Comment:
I am not sure, why we do that, but we are using same convention for all components. I think we can keep this as quo2 for consistency?
https://github.com/status-im/status-mobile/blob/ba1ff91cddcbfbe257283dcd850d22ccb2876e35/src/quo2/screens/buttons/button.cljs#L6

Reply:
Okay, that's new to me because I never name such nested namespaces based on the root (`quo2`), it's usually the leaf `button`). Thanks for clarifying this is already a convention.